.travis.yml

```diff
@@ -9,6 +9,10 @@ env:
   fast_finish: true
 
 install:
+  # The plugin for the setup tools with git and git archives has to be installed manually
+  # in order that as the pip install runs it can do the git discovery for tags etc
+  #
+  - pip install setuptools_scm setuptools_scm_git_archive
```

Comment:
it's a bit odd that you need to install these two packages separately... I mean, I have tried travis with setuptools_scm, and it was working without extra explicit packages (https://github.com/studioml/studio/pull/213/files)

Reply:
But if you are sure these are necessary here, so be it :)